app/src/main/java/com/example/marvelapp/presentation/mvp/presenter/MarvelDetailPresenter.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.marvelapp.presentation.mvp.presenter
+
+import com.example.marvelapp.presentation.mvp.contract.MarvelAppDetailContract
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.schedulers.Schedulers
+
+class MarvelDetailPresenter(
+    private val model: MarvelAppDetailContract.Model,
+    private val view: MarvelAppDetailContract.View
+) : MarvelAppDetailContract.Presenter {
+
+    override fun retrieveSingleCharacterInfo(characterId: Int) {
+        model.getDataSingleCharacter(characterId)
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe(
+                { data ->
+
```

Comment:
Add here a TODO to explain why this is empty

Reply:
Done thx